plasmapy/atomic/tests/test_ionization_states.py

```diff
@@ -735,6 +767,6 @@ def test_equality(self, this, that):
             print(f"{that} kwargs:\n {self.dict_of_kwargs[that]}\n")
             self.instances[that].info()
             descriptor = "equal" if expect_equality else "unequal"
-            raise AtomicError(
+            raise pytest.fail.Exception(
```

Comment:
So I had to check the docs for this...

```
In [2]: pytest.fail.Exception?
Init signature: pytest.fail.Exception(msg=None, pytrace=True)
Docstring:      raised from an explicit call to pytest.fail() 
Type:           type

In [3]: pytest.fail?
Signature: pytest.fail(msg='', pytrace=True)
Docstring:
Explicitly fail an executing test with the given message.

:param str msg: the message to show the user as reason for the failure.
:param bool pytrace: if false the msg represents the full failure information and no
    python traceback will be reported.
File:      ~/.local/lib/python3.7/site-packages/_pytest/outcomes.py
Type:      function

```

So... maybe just this?
```suggestion
            pytest.fail(
```